### PR TITLE
Django1.9

### PR DESCRIPTION
**Pull request by zerc**

### CODE REVIEW

File: tests/functional/test_models.py [hunk 1]
```diff
@@ -114,31 +121,21 @@ class Employee(models.Model):
 class FileGroupFieldTest(unittest.TestCase):
 
     def test_empty_str_is_allowed(self):
-        class Book(models.Model):
-            pages = FileGroupField(blank=True)
         book = Book()
         self.assertEqual(book.pages, '')
```
Comment (homm) [inline tests/functional/test_models.py]: `pages_blank`

File: tests/functional/test_models.py [hunk 2]
```diff
@@ -114,31 +121,21 @@ class Employee(models.Model):
...
 
     def test_null_is_allowed(self):
-        class Book(models.Model):
-            pages = FileGroupField(null=True)
         book = Book(pages=None)
         self.assertEqual(book.pages, None)
```
Comment (homm) [inline tests/functional/test_models.py]: `pages_null`

File: tests/functional/test_models.py [hunk 3]
```diff
@@ -99,12 +112,6 @@ class Employee(models.Model):
     def test_not_found_file_uuid(self, request):
         request.side_effect = InvalidRequestError(404)
 
-        # Just for calling `value.info` inside `Field.clean` method.
```
Comment (homm) [inline tests/functional/test_models.py]: This was just for this test, now for all. It is not clear what is actually happens.

**Comment by zerc** [comment]
Reverted back defining of models but with adding necessary `app_label` for them.

**Comment by homm** [comment]
ok then
:ship:

**Pull request merged**